javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -734,6 +734,42 @@ public static ExecutableElement getMethod(
     }
 
     /**
+     * Returns the ExecutableElement for a method declaration of methodName, in class typeName, with
+     * parameters of the given types.
+     */
+    public static ExecutableElement getMethod(
+            String typeName, String methodName, ProcessingEnvironment env, String... paramTypes) {
+        TypeElement typeElt = env.getElementUtils().getTypeElement(typeName);
+        for (ExecutableElement exec : ElementFilter.methodsIn(typeElt.getEnclosedElements())) {
+            if (exec.getSimpleName().contentEquals(methodName)
+                    && exec.getParameters().size() == paramTypes.length) {
+                boolean typesMatch = true;
+                List<? extends VariableElement> params = exec.getParameters();
+                for (int i = 0; i < paramTypes.length; i++) {
+                    VariableElement ve = params.get(i);
+                    TypeMirror tm = ve.asType();
+                    // TODO: Is there a better way to do this?
```

Comment:
You could use `TypesUtils.isDeclaredOfName(tm, paramTypes[i])` instead of comparing the strings yourself.

Reply:
Sorry, that doesn't work for primitive types. So you either continue doing the string comparison yourself or add a new utility method to `TypesUtils` that works for both declared and primitive types.